packages/elements/tsup.config.ts

```diff
@@ -1,8 +1,27 @@
+import type { Plugin } from 'esbuild';
 import { defineConfig } from 'tsup';
 
 import { version as clerkJsVersion } from '../clerk-js/package.json';
 import { name, version } from './package.json';
 
+/**
+ * Replaces the inspect with undefined so-as not to bundle
+ * inspectors outside of our development environment.
+ */
+export function dynamicInspectorImport(): Plugin {
```

Comment:
Converts exported inspect into `undefined` which is handled appropriately by xstate.